kebechet/cli.py

```diff
@@ -88,7 +90,24 @@ def cli_run_results(origin, service, analysis_id):
 @click.option("-s", "--service", envvar="KEBECHET_SERVICE_NAME")
 def cli_run_url(url, service):
     """Run Kebechet by providing url to a git repository service."""
-    config.run_url(url, service)
+    config.run_url(url, service, None)
+
+
+@cli.command("run-webhook")
+@click.argument("web_payload", nargs=1)
+def cli_run_webhook(web_payload):
+    """Run Kebechet by providing a webhook payload."""
+    payload = None
+    if os.path.isfile(web_payload):
+        with open(web_payload) as f:
+            payload = json.load(f)
+    else:
+        # If the json is passed a string.
+        payload = json.loads(web_payload)
+    if payload:
+        config.run_webhook(payload)
+    else:
+        _LOGGER.exception("Payload passed to webhook handler is empty.")
```

Comment:
`_LOGGER.exception` logs traceback and should be run within exception handling context (in try-except block).

To raise an exception, write something like:

```python
    if not payload:
        raise SomeExceptionType("Payload passed to webhook handler is empty.")
```

Reply:
`if not payload:
        raise WebhookPayloadError("Webhook payload is empty or cannot be parsed.")
    config.run_webhook(payload)`
Something like this?

Reply:
:+1: